docs/DeveloperGuide.md

```diff
@@ -2,12 +2,153 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
+original source as well}
+
+[Original Source](https://github.com/nus-cs2113-AY2425S1/tp)
 
 ## Design & implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+## Overall Class Diagram
+![overallClass.jpg](./Diagrams/overallClass.jpg)
```

Comment:
![image](https://github.com/user-attachments/assets/977a8e4b-32c3-4355-aceb-db0d30594f8b)
Is this the right way to add comments to your class diagram? You might want to put the note outside the box and then have a dotted line pointing to the note from the box